crates/bevy_sprite/src/sprite.rs

```diff
@@ -16,4 +16,48 @@ pub struct Sprite {
     /// An optional custom size for the sprite that will be used when rendering, instead of the size
     /// of the sprite's image
     pub custom_size: Option<Vec2>,
+    /// [`Anchor`] point of the sprite in the world
+    pub anchor: Anchor,
+}
+
+/// How a sprite is positioned relative to its [`Transform`](bevy_transform::components::Transform).
+/// It defaults to `Anchor::Center`.
+#[derive(Debug, Clone, Reflect)]
+#[doc(alias = "pivot")]
+pub enum Anchor {
+    Center,
+    BottomLeft,
+    BottomCenter,
+    BottomRight,
+    CenterLeft,
+    CenterRight,
+    TopLeft,
+    TopCenter,
+    TopRight,
+    /// Custom anchor point. Top left is `(-0.5, 0.5)`, center is `(0.0, 0.0)`. The value will
+    /// be scaled with the sprite size.
```

Comment:
Just a thought, but should this be (0, 0) bottom left to (1, 1) top right? I think that might be clearer for those who want to select an anchor as an absolute position using `Custom` rather than as an offset from the center point (which could also maybe play into UI stuff as well).

Reply:
I like that (0, 0) is the default behavior personally.

It doesn't matter much as long as it's clearly documented though.

Reply:
The default being centered is good in my opinion as well.